openapi/external_api.yaml

```diff
@@ -58,6 +58,34 @@ paths:
                 $ref: '#/components/schemas/HTTPErrorResponse'
         '500':
           $ref: '#/components/responses/InternalError'
+  /operations/{schedulingID}/{correlationID}/debug:
```

Comment:
I think it will be better if you set debug mode as parameter in order to follow REST practices:
/operations/{schedulingID}/{correlationID}/**?debug=1**
The default value is 0, What do you think?

Reply:
That's a good idea. But I am not sure if this applies in this case. Because, debug as a url param seems to me like I can disable debug looging for an operation or a reconciliation, which is not true and not needed by this feature because the operation will run once and the next operation debug logs will need to be enabled separately.

Reply:
Hmm, understood, then it make sense to not change it then :)